app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailInputFilter.java

```diff
@@ -0,0 +1,76 @@
+package fr.free.nrw.commons.upload;
+
+import android.text.InputFilter;
+import android.text.Spanned;
+import java.util.regex.Pattern;
+
+/**
+ * An {@link InputFilter} class that removes characters blacklisted by Wikimedia. The list of
```

Comment:
Would you mind replacing `characters blacklisted by Wikimedia` with `characters blocklisted by Wikimedia for titles`?
- `blocklisted` to avoid reinforcing white/black=good/bad vocabulary (even though Wikimedia has not been amended this yet, our app is usually leading by example)
- `for titles` because such characters are allowed in other parts of Wikimedia.